bitgouel-api/src/main/kotlin/team/msg/domain/university/mapper/UniversityMapper.kt

```diff
@@ -0,0 +1,8 @@
+package team.msg.domain.university.mapper
+
+import team.msg.domain.university.presentation.data.request.CreateUniversityRequest
+import team.msg.domain.university.presentation.data.web.CreateUniversityWebRequest
+
+interface UniversityMapper {
```

Comment:
```suggestion
interface UniversityRequestMapper {
```
UniversityMapperImpl -> UniversityRequestMapperImpl 변경해 주세요!